contestant_containers/arch/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/archlinux
+LABEL maintainer "Trevor Bryant <@apporima>"
+
+# WORKAROUND for glibc 2.33 and old Docker
+# See https://github.com/actions/virtual-environments/issues/2658
+# Thanks to https://github.com/lxqt/lxqt-panel/pull/1562
+RUN patched_glibc=glibc-linux4-2.33-4-x86_64.pkg.tar.zst && \
+    curl -LO "https://repo.archlinuxcn.org/x86_64/$patched_glibc" && \
+    bsdtar -C / -xvf "$patched_glibc"
+
+# Install packages
+RUN pacman -Sy --noconfirm git make gcc pkgconf which python-pip wget hostapd wpa_supplicant bully openssh vim wireshark-cli tmux screen supervisor
+
+# Download current WCTF wordlist
+RUN wget https://raw.githubusercontent.com/rfhs/wctf-files/master/wordlist/cyberpunk.words -O /root/cyberpunk.words
+
+# Permit root login
+RUN sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
```

Comment:
This is almost certainly not desirable for anyone.  If you are using it locally just exec /bin/bash and if you are using it remotely then you don't want anyone else logging in.

Reply:
Agreed. These lines won't exist when you port testing over to your AWS environment. In fact, I can remove these now if you would prefer.

Reply:
this is a pull request, so please :-)  I intend this to be usable by anyone with minimal modification, but this is not the kind of example I wish to set.